p4-16/spec/P4-16-spec.mdk

```diff
@@ -2207,9 +2207,13 @@ target-specific).
 It is also possible to specify an `enum` with an underlying representation.
 These are sometimes called serializable `enum`s, because headers are
 allowed to have fields with such `enum` types.
-This requires the programmer provide both the fixed-width unsigned (or signed) integer type and
-an associated integer value for each symbolic entry in the enumeration.  For example, the
```

Comment:
It seems good to me to keep this sentence in the spec as it was before: "This requires the programmer provide both the fixed-width unsigned (or signed) integer type and an associated integer value for each symbolic entry in the enumeration."

It is not redundant with the new text being added -- they both say different things.